### DISCUSSION TITLE
GitHub Mobile demands at least one shortcut to remove the huge reminder

### DISCUSSION THREAD

**Discussion by Corionis** [community/community#64868] [Mobile]
### Select Topic Area

Bug

### Body

Provide a means to remove the Shortcuts (and Favorites) from GitHub Mobile and have no shortcuts.  Also, there is no support for viewing Milestones which we use to order (sequence) our work.

**Comment by Jaber-Saed** [comment]
The issue at hand pertains to GitHub Mobile and encompasses both a bug and a feature request for improvement. Users are facing difficulties in managing shortcuts and favorites within the GitHub Mobile app. Currently, there is no functionality that allows users to remove shortcuts or favorites from the app, leaving them with an array of shortcuts that might no longer align with their preferences or usage patterns. This issue impacts the overall user experience by cluttering the interface and impeding quick navigation, ultimately hindering efficient interaction with the app.

Furthermore, there is a notable feature omission in GitHub Mobile: the absence of the ability to view milestones. Milestones play a pivotal role in project management, aiding in the organization and sequencing of tasks and goals. However, due to the absence of this feature in the mobile app, users are unable to effectively track progress, plan workflows, and gauge project advancement.

To address these concerns and enhance the usability of GitHub Mobile, a few potential solutions could be considered:

Shortcuts and Favorites Management: Introducing the capability to manage shortcuts and favorites within the app would grant users the freedom to customize their app interface. This could involve the option to add, remove, or rearrange shortcuts and favorites according to individual preferences and requirements.

Milestone Viewing: Incorporating support for viewing milestones in GitHub Mobile is crucial for project management continuity. Enabling users to access milestone details, monitor progress, and gain insights into related issues or pull requests directly from the mobile app would empower them to efficiently manage their projects while on the go.

By addressing the reported bug regarding shortcuts and favorites, and by implementing the requested milestone viewing feature, GitHub Mobile could be significantly improved for a broader user base. Enhancing the flexibility and functionality of the app aligns with the overarching goal of offering a seamless and adaptable experience to users engaging with GitHub on mobile devices.
